src/server/ws-routes/initializeWebSocketRouting.js

```diff
@@ -0,0 +1,10 @@
+const ConnectionsRepository = require('./ConnectionsRepository')
+
+const Utilities = require('../utilities')
+
+
+module.exports = initializeWebSocketRouting
+
+function initializeWebSocketRouting(io, protocol = require('../protocol/protocol.json')) {
+	Utilities.createWebsocketRoute(io, protocol.room, require('./controllers/Room'), new ConnectionsRepository())
```

Comment:
For me `require('./controllers/Room')` is unclear.  What this argument do ? What is that ???!1!11!!

Reply:
It's routing but for web sockets. Think of it like Router from express.
We'll place here every new ws endpoint in this pattern. I can declare it at the beginning of the file, if you wish.